packages/scene-composer/tests/components/three-fiber/EditorCamera.spec.tsx

```diff
@@ -70,6 +79,11 @@ import { EditorMainCamera, findBestViewingPosition } from '../../../src/componen
 import { useStore } from '../../../src/store';
 
 import { useThree } from '@react-three/fiber';
+
+import { OrbitControls } from '../../../src/three/OrbitControls';
+//auto mock for instanceof check passing
+//jest.mock('../../../src/three/OrbitControls');
```

Comment:
Nit-pick: Should probably just delete unless there's a TODO associated with these. If that's the case, would like to see the TODO comment inline with these.

Reply:
Removed